packages/types/src/admin-meta.ts

```diff
@@ -1,7 +1,34 @@
+import { GraphQLError } from 'graphql';
 import type { ReactElement } from 'react';
 import { GqlNames, JSONValue } from './utils';
 
-export type AdminConfig = { components?: { Logo?: (props: {}) => ReactElement } };
+export type NavigationProps = {
```

Comment:
@mitchellhamilton  impetus here is to avoid circular dependency of `@keystone-next/types` on `@keystone-next/keystone` 

Chatted with @JedWatson  about this beforehand, but happy to revert or change as necessary. 
Also overall keen to understand the boundaries around @keystone-next/types, what should go in here and what shouldn't etc.